src/dotnet/install.sh

```diff
@@ -230,6 +230,21 @@ install_using_apt() {
     fi
 }
 
+install_using_default_apt_repo() {    
+    apt_get_update_if_needed  
+    DOTNET_PACKAGE="dotnet6"
+
+    if [ "${DOTNET_VERSION}" = "latest" ] || [ "${DOTNET_VERSION}" = "lts" ] || "${DOTNET_VERSION}" = "6.0"; then
```

Comment:
> "${DOTNET_VERSION}" = "6.0"

Should we instead check for the major version (`6`)?

Reply:
I was mirroring what we had in the `devcontainer-feature.json`, but I agree it makes more sense to present major versions, so changed that.